src/lib/parseArgs/paParse.cpp

```diff
@@ -565,22 +565,11 @@ int paParse
     RETURN_ERROR("paOptionsParse");
   }
 
-  if (paLogSetup() == -1)
-  {
-    RETURN_ERROR("paLogSetup error");
-  }
-
   if ((s != -2) && ((s = paLimitCheck(paiList)) == -1))
   {
     RETURN_ERROR("paLimitCheck");
   }
 
-  if ((s != -2) && (paConfigActions(false) == -1))
-  {
-    RETURN_ERROR("paConfigActions");
-  }
-
-
```

Comment:
We this removal is needed? Could you elaborate?

Reply:
I moved both methods to the contextBroker main method, because in the previous order the log-system was setup before the env variables where evaluated. Therefor the -fg, -logForHumans and the new -disableFileLog options where only available as commandline options. With extracting paLogSetup and paConfigActions from the parse method, its now possible to parse the options and configure the logging based on the results of the parsing. See contextBroker.pp l953 - l988